src/com/oltpbenchmark/benchmarks/twitter/ddls/twitter-nuodb-ddl.sql

```diff
@@ -15,15 +15,15 @@ CREATE INDEX IDX_USER_PARTITION ON user_profiles (partitionid);
 
 DROP TABLE IF EXISTS followers CASCADE;
 CREATE TABLE followers (
-  f1 int NOT NULL,
-  f2 int NOT NULL,
+  f1 int NOT NULL REFERENCES user_profiles (uid),
```

Comment:
NuoDB might not support this. You can leave it out

Reply:
To double-check, you mean NuoDB may not support having a primary key that is composed of two foreign keys, and so it should be
f1 int not NULL
for both followers and follows, but f2 is fine?

Reply:
Actually, according to their documentation, they should support `REFERENCES`

http://doc.nuodb.com/Latest/Content/CREATE-TABLE.htm

You can keep this.